docs/guides/lists.md

```diff
@@ -12,23 +12,24 @@ more in-depth in this guide, but first, let's cover how to use lists.
 
 ### Using Lists
 
-To use a list, add the `list` property to your
-[CRUD operation actions](./crud-actions.md).
+To use a list, add the `list` property to a success
+[CRUD operation action](./crud-actions.md).
 
 ```js
 import { actionTypes } fom 'redux-resource';
 import store from './store';
 
 store.dispatch({
-  type: actionTypes.READ_RESOURCES_PENDING,
+  type: actionTypes.READ_RESOURCES_SUCCEEDED,
   resourceName: 'books',
   list: 'mostPopular',
-  // Remember, we always want to track the request status
-  request: 'getMostPopular'
```

Comment:
We should keep the request on the action, I think. We may still need it to show a success message somewhere!